.github/workflows/sql-benchmarks.yml

```diff
@@ -23,15 +23,18 @@ jobs:
             binary_name: tpch
             name: TPC-H on NVME
             local_dir: bench-vortex/data/tpch/1
+            targets: "datafusion:arrow,datafusion:parquet,datafusion:vortex,duckdb:parquet,duckdb:vortex"
           - id: clickbench-nvme
             binary_name: clickbench
             name: Clickbench on NVME
             local_dir: bench-vortex/data/clickbench_partitioned
+            targets: "datafusion:parquet,datafusion:vortex,duckdb:parquet,duckdb:vortex"
```

Comment:
```suggestion
            targets: "datafusion:parquet,datafusion:vortex,duckdb:parquet,duckdb:vortex,duckdb:duckdb"
```

Reply:
I thought you said we should close this PR

Reply:
I think its a good idea to be explicit

Reply:
I think we should merge this